app/Http/Controllers/Apis/Protected/Summit/Factories/SummitAbstractLocationValidationRulesFactory.php

```diff
@@ -27,17 +27,21 @@ public static function build(array $data, $update = false){
 
         if($update){
             return [
-                'name'        => 'sometimes|string|max:255',
-                'short_name'        => 'sometimes|string|max:255',
-                'description' => 'sometimes|string',
-                'order'       => 'sometimes|integer|min:1'
+                'name'         => 'sometimes|string|max:255',
+                'short_name'   => 'sometimes|string|max:255',
+                'description'  => 'sometimes|string',
+                'opening_hour' => 'sometimes|integer|min:0|max:2359',
+                'closing_hour' => 'sometimes|integer|gt:opening_hour|min:0|max:2359',
```

Comment:
you need https://laravel.com/docs/10.x/validation#rule-required-with here 
like required_with:opening_hour

Reply:
done